sdk/storage/Azure.Storage.Files.DataLake/perf/Azure.Storage.Files.DataLake.Perf/Azure.Storage.Files.DataLake.Perf.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyName>Azure.Storage.Files.DataLake.Perf</AssemblyName>
```

Comment:
I think for SDK-style projects, assembly name and root namespace are the project name, unless overridden.
Since it is the same as the project name here, I think this property can be skipped.

Reply:
Nice catch.   I didn't add that explicitly, not sure how that snuck in there.